README.md

```diff
@@ -125,7 +125,7 @@ The color used for the track to the left of the button. Overrides the default bl
 
 | Type               | Required |
 | ------------------ | -------- |
-| [color](colors.md) | No       |
+| [color](https://github.com/facebook/react-native-website/blob/master/docs/colors.md) | No       |
```

Comment:
It would be nice if it links to the actual page in the document website
```suggestion
| [color](https://reactnative.dev/docs/colors) | No       |
```